azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/AzureADGraphClient.java

```diff
@@ -46,4 +80,73 @@ private static String getResponseStringFromConn(HttpURLConnection conn) throws I
             return stringBuffer.toString();
         }
     }
+
+    public List<UserGroup> getGroups(String graphApiToken) throws IOException {
+        return loadUserGroups(graphApiToken);
+    }
+
+    private List<UserGroup> loadUserGroups(String graphApiToken) throws IOException {
+        final String responseInJson = getUserMembershipsV1(graphApiToken);
+        final List<UserGroup> lUserGroups = new ArrayList<>();
+        final ObjectMapper objectMapper = JacksonObjectMapperFactory.getInstance();
+        final JsonNode rootNode = objectMapper.readValue(responseInJson, JsonNode.class);
+        final JsonNode valuesNode = rootNode.get("value");
+        int i = 0;
+        while (valuesNode != null
```

Comment:
seems i only appears in loop, how about `for (int i; condition; i++)`

Reply:
Thanks for the remind, will update.